scenario/elasticexplore.go

```diff
@@ -373,7 +373,7 @@ func searchForTag(sessionState *session.State, actionState *action.State, host,
 		return id, nil
 	}
 
-	query := fmt.Sprintf("%s/api/v1/collections?query=%s&type=public&-updateAt", host, collection)
+	query := fmt.Sprintf("%s/api/v1/collections?query=%s&limit=20&sort=-name&type=public", host, collection)
```

Comment:
When query gets re-assigned at line 392 (in case of paging) from the links objects in the response, does this query already follow the correct format that client would issue?

Reply:
I started checking and realized that might be quite the corner case. Since the parameter query=%s is set to the collection name, this case will only be hit if there are more than 20 collections of the matching name.